src/opers.h

```diff
@@ -221,17 +222,35 @@ static inline void SET_ENABLED_ATTR(Obj oper, Int x)
 /****************************************************************************
 **
 *F * * * * * * * * * * * * internal flags functions * * * * * * * * * * * * *
+**
+** Attempting change April 2018. Flags will be stored as 
+** AND_CACHE -- Obj
+** Size (UInt)  -- could possibly be a UInt2 instead
+** Hash (Uint)
+** Set positions (array of UInt2).
```

Comment:
What does `Set positions` mean? `Set` as in the verb? Or the mathematical object? also, how are they stored? In ascending order?

Reply:
Will imptove comment